core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -53,13 +53,13 @@ public static UserRepository getRepository()
 
 	private boolean isCreatedOnBuddyRequest;
 
-	@OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER, orphanRemoval = true)
+	@OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY, orphanRemoval = true)
```

Comment:
This is the default, so does not have to be here, see [Hibernate docs](http://docs.jboss.org/hibernate/core/4.3/manual/en-US/html/ch20.html#performance-fetching-lazy).